modules/cloudbuild/cloudbuild_builder/Dockerfile

```diff
@@ -32,7 +32,9 @@ RUN apt-get update && \
     sha256sum -c terraform_SHA256SUMS --status && \
     unzip terraform_linux_amd64.zip -d /builder/terraform && \
     rm -f terraform_linux_amd64.zip && \
-    gsutil cp gs://terraform-validator/releases/${ENV_TERRAFORM_VALIDATOR_RELEASE}/terraform-validator-linux-amd64 /builder/terraform/terraform-validator && \
+    TERRAFORM_VALIDATOR_VERSION=$(echo $ENV_TERRAFORM_VALIDATOR_RELEASE | sed -e 's/^v//') && \
+    TERRAFORM_VALIDATOR_GCS_PATH=gs://terraform-validator/releases/${ENV_TERRAFORM_VALIDATOR_RELEASE}/terraform-validator_linux_amd64-${TERRAFORM_VALIDATOR_VERSION}.tar.gz && \
+    gsutil cat "${TERRAFORM_VALIDATOR_GCS_PATH}" | tar -zxv -C /builder/terraform terraform-validator && \
```

Comment:
Can we use `cp` than `cat`? It seems like cat does not validate checksum https://cloud.google.com/storage/docs/gsutil/commands/cat#:~:text=output%20to%20stdout.)-,warning,-%3A%20The%20gsutil%20cat

Reply:
Done.

Also updated Terraform to 0.13.7 - latest in 0.13. No real changes there, just [public key update](https://github.com/hashicorp/terraform/blob/v0.13/CHANGELOG.md)

Reply:
@bharathkkb PR review comments addressed.